src/dsp_decaytime.jl

```diff
@@ -7,10 +7,10 @@ Get statistics on the logarhithmic of the tail of the `wvfs` in the interval (`s
 # Returns
 - `τ`: decay time in µs
 """
-function dsp_decay_times(wvfs::ArrayOfRDWaveforms, config::DSPConfig)
+function dsp_decay_times(wvfs::ArrayOfRDWaveforms, bl_mean::Tuple, pz_fit::Tuple)
```

Comment:
`bl_mean` is a window, right? People might misunderstand it to mean "baseline-level". I think we should use something like `blwindow::AbstractInterval{<:RealQuantity}` instead.

Maybe `pz_fit` should also be an interval?

Reply:
Both `bl_mean` and `pz_fit` are intervals. I did not introduce this, this is how it was in the original code - `config.bl_mean` and `config.pz_fit`. I simply introduced a function that takes these arguments directly rather than through the `DSPConfig` struct so that it's more accessible.

So the renaming that you suggest is a parallel modification to the one in this PR.

If you and the team agree, I could implement it in this PR together with my change. However, this would slow down this PR, and my point here was to make a small change just so that I can share my code (where this change was in local dev) with colleagues so they can use it.

Also that would require more changes in the `DSPConfig` type in addition to the function I touched, so needs to be more careful and is probably better for someone more familiar with the code than me, in order to not break any routines exactly as @theHenks pointed out. So I would vote for merging this PR if you confirm this is just a small multiple dispatch addition and does not break routines; and proceeding with variable name changes in a different PR.

Reply:
Maybe we can also rethink the whole structure of the `DSPConfig` struct. My idea here was to have one struct which is the only metadata input to any function in the DSP package, so that one never has to worry about it anymore. 
I am happy to rename and change types. It is just that the current names also correspond to the names of the variables in the `legend-metadata` jl dataflow config. So if we change that I would like to also change that there.

Reply:
The Intervals shouldn't be a problem. We can merge that here. I would just like then to have it working with the `create_dsp_config` function to set up the struct. So I suggest to rewrite that and then include a test with an example metadata so that we are sure nothing breaks.

Reply:
Again, renaming variables for clarity I think belongs in a different PR, as it has nothing to do with the small addition that I made, and is more of a general package development. It would be convenient to create an issue and collect all these items (docstrings, meaningful variable names), and then proceed to do a PR based on the issue.

Reply:
I chanegd it in the main already to be intervals. Works now also seamlessly with the unit reading and parsing from the metadata files. Can be merged now.